iceoryx2-cal/src/static_storage/file.rs

```diff
@@ -253,24 +253,36 @@ impl crate::named_concept::NamedConceptMgmt for Storage {
 
         let adjusted_path = config.path_for(storage_name);
 
-        match File::does_exist(&adjusted_path) {
-            Ok(true) => (),
-            Ok(false) => return Ok(false),
+        let does_exist = || match File::does_exist(&adjusted_path) {
+            Ok(true) => Ok(true),
+            Ok(false) => Ok(false),
             Err(v) => {
                 fail!(from origin, with NamedConceptDoesExistError::UnderlyingResourcesCorrupted,
                     "{} due to an internal failure ({:?}), is the static storage in a corrupted state?", msg, v);
             }
         };
```

Comment:
This could be done with a `map_err` instead of a full match.

Reply:
`or_else` also works ... I will use this instead.

Reply:
Slightly less idiomatic but works with the macro :)